plugins/klock.yaml

```
@@ -77,5 +116,7 @@ spec:
 
   shortDescription: Watches resources
   description: |
-    Watches resources and prints them in a human-readable manner.
-    Very similar to `kubectl get`, but with live updates.
+    Usage:
+      $ kubectl klock <resource> [name]
+      This plugin watches resources and prints them in a human-readable manner. Very similar to `kubectl get`, but with live updates.
+      Read more documentation at: https://github.com/jilleJr/kubectl-klock
```

Oops, forgot to update the description in my `.krew.yaml`

```suggestion
    Watches resources and prints them in a human-readable manner.
    Very similar to `kubectl get`, but with live updates.
```